spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'David', photo: 'http://', bio: 'Teacher from Mexico') }
+
+  before { subject.save }
+
+  it 'title should be present' do
```

Comment:
- [OPTIONAL] I don't think you are testing for `title` attribute here, rather you are testing for the `name` attribute. I would recommend you fix this as well